contracts/StablecoinConverter.sol

```diff
@@ -133,9 +133,11 @@ contract StablecoinConverter is EpochTokenLocker {
             batchIndex == getCurrentStateIndex() - 1,
             "Solutions are no longer accepted for this batch"
         );
+        require(checkPriceOrdering(tokenIdsForPrice), "prices are not ordered by tokenId");
```

Comment:
Why is this needed? I believe otherwise the binary search would not work, but wouldn't the function revert in that case (thus it's in the submitters best interest to sort it)?

Reply:
If tokenIdsForPrice references to different prices for the same tokens, then the oracle function might not work properly., i.e tokenIdsForPrice[0,1,0] provides two prices for token 0.